### ISSUE TITLE
BBW-Chan Issues

### ISSUE THREAD

**Issue by CodeMonkeyAlec** [forbjok/chandler2#1]
I am trying to use the app to download from bbw-chan.net and no matter what I do I get the error message
## `object.Error@(0): Access Violation

0x0044CE46
0x004026EA
0x0047E8D3
0x0047E897
0x0047E798
0x0042A218
0x75F9EF1C in BaseThreadInitThunk
0x771F3648 in RtlInitializeExceptionChain
0x771F361B in RtlInitializeExceptionChain`

that seems to be the issue and been bagging my head off my desk trying to figure out what's wrong, I hope you can maybe help

I have tried on both my PC and Linux Computer

**Comment by forbjok** [comment]
There was an issue with hyphens in hostnames. It should be fixed now in v2.0.0-beta3.

**Comment by CodeMonkeyAlec** [comment]
hey tried using it with the new url for the site bbw-chan.nl

got this error
```
std.net.curl.CurlException@std\net\curl.d(3894): Failed to load curl, tried "libcurl.dll", "curl.dll".
----------------
0x0041804E
0x004927CD
0x0043DD27
```

Not sure what to do?
